docs/src/usage/configuration.md

```diff
@@ -1,7 +1,7 @@
 # Configuration
 selene is meant to be easily configurable. You can specify configurations for the entire project as well as for individual lints.
 
-Configuration files are placed in the directory you are running selene in and are named **selene.toml**. As the name suggests, the configurations use the [Tom's Obvious, Minimal Language (TOML)](https://github.com/toml-lang/toml) format. It is recommended you quickly brush up on the syntax, though it is very easy.
+Configuration files apply to the directory (and subdirectories) in which they are placed. So, to control how selene behaves when it lints scripts in, say, `~/myScripts` and below, place the configuration file within (the top-level of) `~/myScripts`. Each configuration file is to be called **selene.toml**. As the file suffix suggests, configuration files use the [Tom's Obvious, Minimal Language (TOML)](https://github.com/toml-lang/toml) format. It is recommended you quickly brush up on the syntax, though it is very easy.
```

Comment:
> Configuration files apply to the directory (and subdirectories) in which they are placed. So, to control how selene behaves when it lints scripts in, say, `~/myScripts` and below, place the configuration file within (the top-level of) `~/myScripts`.

This isn't accurate because the config doesn't apply to the scripts in the folder. It applies to the command when ran from the folder, which can lint on a different folder.